Drivers/CMSIS/Device/ST/STM32F4xx/Include/stm32f4xx.h

```diff
@@ -73,7 +73,7 @@
   /* #define STM32F439xx */   /*!< STM32F439VG, STM32F439VI, STM32F439ZG, STM32F439ZI, STM32F439BG, STM32F439BI, STM32F439NG, 
                                    STM32F439NI, STM32F439IG and STM32F439II Devices */
   /* #define STM32F401xC */   /*!< STM32F401CB, STM32F401CC, STM32F401RB, STM32F401RC, STM32F401VB and STM32F401VC Devices */
-  /* #define STM32F401xE */   /*!< STM32F401CD, STM32F401RD, STM32F401VD, STM32F401CE, STM32F401RE and STM32F401VE Devices */
+   #define STM32F401xE    /*!< STM32F401CD, STM32F401RD, STM32F401VD, STM32F401CE, STM32F401RE and STM32F401VE Devices */
```

Comment:
This file should not be modified, instead add a build configurations (debug and release) and set the symbol there. Right click on the project and select _Build configurations > Manage..._ then _New..._ Copy the settings from F401 Blackpill and change the mcu symbol afterwards.